### ISSUE TITLE
Can't use with storyboard

### ISSUE THREAD

**Issue by EternalBlack** [mchoe/SwiftSVG#44]
As the title tells, I'm not able to make it work with storyboards. Entering `SVGView` to a UIView doesn't present me an option to enter the SVGName. It simply doesn't detect/recognize the class. Also it's not available in the dropdown. Settings the `module` manually also doesn't help.

Doeing this via code i.e. `import` etc works fine, just storyboard doesn't work.

Installation was via carthage so maybe this is related to #27

**Comment by mchoe** [comment]
Hi @EternalBlack , can you please try this with the latest version 2.0.0? Thanks.

**Comment by mchoe** [comment]
Seems to be related to https://github.com/Carthage/Carthage/issues/335

**Comment by jsahoo** [comment]
Agreed, this does seem to be related to https://github.com/Carthage/Carthage/issues/335. I was able to fix the same issue for Cosmos https://github.com/evgenyneu/Cosmos/pull/110 - presumably the same fix should work for SwiftSVG.
